### PR TITLE
Lime 648 cleanup of post merge int tests

### PR DESCRIPTION
**Pull request by chinmayi-gds**
### What changed
The acceptance tests that were lifted from ‘ipv-core-tests’ repo were compared to what we have in the ‘di-ipv-cri-uk-passport-back’ repo and the duplicates have been removed. The tests which are required and were not previously available in ‘di-ipv-cri-uk-passport-back’ repo have been retained. The test runs have all passed.
Please note that the accessibility tests that have been lifted from ‘ipv-core-tests’ repo on to ‘di-ipv-cri-uk-passport-back’ repo are currently not functional. These tests will have to be moved to the new passport CRI Front repo and the work to fix these tests will be done as part of a different ticket.

### Issue tracking

- [PYI-XXXX](https://govukverify.atlassian.net/browse/PYI-XXX)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

### CODE REVIEW

File: acceptance-tests/src/test/java/uk/gov/di/ipv/cri/passport/acceptance_tests/pages/PagesFromIpvCoreRepo/FraudCheckStubPage.java [hunk 1]
```diff
@@ -27,27 +27,6 @@ public FraudCheckStubPage() {
...
-    @FindBy(id = "requested_oauth_error_endpoint_2")
-    public WebElement AccessToken;
-
-    @FindBy(id = "requested_oauth_error")
-    public WebElement OAuthError;
-
-    @FindBy(id = "requested_oauth_error_description")
-    public WebElement OAuthErrorDescription;
-
     public static void enterFraudDetails() {
```
Comment (gds-jbentham) [inline acceptance-tests/src/test/java/uk/gov/di/ipv/cri/passport/acceptance_tests/pages/PagesFromIpvCoreRepo/FraudCheckStubPage.java:30]: Are we actually using this page?

File: acceptance-tests/src/test/java/uk/gov/di/ipv/cri/passport/acceptance_tests/pages/PagesFromIpvCoreRepo/IpvCoreFrontPageArchive.java [hunk 2]
```diff
@@ -7,16 +7,16 @@
 
 public class IpvCoreFrontPageArchive {
 
+    @FindBy(xpath = "//*[@class='govuk-heading-xl']")
```
Comment (gds-jbentham) [inline acceptance-tests/src/test/java/uk/gov/di/ipv/cri/passport/acceptance_tests/pages/PagesFromIpvCoreRepo/IpvCoreFrontPageArchive.java:10]: This file is named archive are we still using?

File: acceptance-tests/src/test/java/uk/gov/di/ipv/cri/passport/acceptance_tests/pages/PagesFromIpvCoreRepo/PassportPage.java [hunk 3]
```diff
@@ -6,65 +6,9 @@
 import uk.gov.di.ipv.cri.passport.acceptance_tests.utilities.Driver;
 
 public class PassportPage {
-
```
Comment (gds-jbentham) [inline acceptance-tests/src/test/java/uk/gov/di/ipv/cri/passport/acceptance_tests/pages/PagesFromIpvCoreRepo/PassportPage.java:9]: Is this page still being used?

File: acceptance-tests/src/test/java/uk/gov/di/ipv/cri/passport/acceptance_tests/pages/PagesFromIpvCoreRepo/FraudCheckStubPage.java (hunk 1 above)
Comment (chinmayi-gds) [reply acceptance-tests/src/test/java/uk/gov/di/ipv/cri/passport/acceptance_tests/pages/PagesFromIpvCoreRepo/FraudCheckStubPage.java:30]: Hi @gds-jbentham, yes, these are being used by the accessibilty tests which we have copied over from the ipv-core-tests repo. These are currently not working but will be fixed as part of the ticket https://govukverify.atlassian.net/browse/LIME-625

**Review by gds-jbentham — APPROVED** [review]

**Pull request merged**